variants/DISCO_F100RB/variant.h

```diff
@@ -132,16 +132,7 @@ enum {
 #define USER_BTN                5
 
 //SPI definitions
-//define 16 channels. As many channel as digital IOs
-#define SPI_CHANNELS_NUM        16
-
-//default chip salect pin
-#define BOARD_SPI_DEFAULT_SS    43
-
-//In case SPI CS channel is not used we define a default one
-#define BOARD_SPI_OWN_SS        SPI_CHANNELS_NUM
-
-#define SS                      BOARD_SPI_DEFAULT_SS
+#define SS                      43
```

Comment:
Not related to this commit, but was is 43 ? can't we use a pin name like e.g. D10 or PB13 ?

Reply:
43 == D43

Reply:
See PR #83 for more information.

Reply:
I understand that D43 is same as 43 and same as PX_y ...but D43 or 43 cannot be read anywhere on my board schematics or on the board itself, so I would prefer to use a commonly used names (either  D0/D16 A0/A5 or PX_y ....

Reply:
Agree that on VLdisco there is no Arduino connector and so using Dx is not well appropriate while all PXn names are on the board silkscreen print. But it's not the subject of this PR. Currently, variant.h using the same naming (Dx). Probably a new PR could be done to update all variants which do not have Arduino connector or those one using a pin number > 15.

Reply:
I agree, that's why I started my comment with "Not related to this commit" ;-)